dask_image/ndinterp/__init__.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+
+
+import numpy as np
+import dask.array as da
+from scipy import ndimage
+
+
+__all__ = [
+    "affine_transform",
+]
+
+
+def affine_transform(
+        input,
```

Comment:
One slight difference between dask-image and scipy is that we use `image` instead of `input`
https://github.com/dask/dask-image/pull/117

Reply:
That's better, changed!